src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/OnCreated

```diff
@@ -13,4 +13,4 @@ try:
     node.parm('queue').set(queue_response['displayName'])
     node.hdaModule().update_queue_parameters_callback(kwargs)
 except CredentialRetrievalError:
-    print('Expired credentials')
\ No newline at end of file
+    print('AWS Deadline Cloud credentials are expired. Please open the Deadline Cloud monitor and log in.')
```

Comment:
I like the clearer error message that it's the credentials for AWS Deadline Cloud that are expired. I don't know if we should be pointing directly to the monitor to login since you can use standard AWS profiles and credential processes to authenticate as well which you don't need to go to the monitor to refresh or configure.

Reply:
Great point! Updated to just "AWS Deadline Cloud credentials are expired" for this PR. We could make the message more actionable like we do for the [Unreal engine plugin](https://github.com/aws-deadline/deadline-cloud-for-unreal-engine/blob/bf6ae92a303a2f5b57fafef7a8a34939257a99bf/src/unreal_plugin/Content/Python/settings.py#L358), but there would need to be more logic in the OnCreated file and I'm not sure if that's plumbed through. Created a ticket to address this: https://github.com/aws-deadline/deadline-cloud-for-houdini/issues/134